### PR TITLE
Rats eyes glow in dark + reorganize rat textures 

### PR DESCRIPTION
**Pull request by jamessimo**
## About the PR

Gave RatKing, BuffRat and Rats unshaded eyes (like spacecarps) so they can be seen in the dark.
Split rats from mice in texture resources, no longer hardcode a mouse variant as a rat.
Fixed misleading admin smite text to turn player into a mouse. 

## Why / Balance

Balance so its easier to ID that rats are a hostile mob 
Why because it looks rad. 

## Technical details

Rats used to share there sprites with mice and had a hardcoded mouse-3 sprite, put that in its own rat.rsi allowing for multi rat variants if in future. 

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame

RATS

## Breaking changes

N/A

**Changelog**

:cl:
- tweak: Ratkings and Rat servants eyes now glow in the dark

### CODE REVIEW

File: Resources/Locale/en-US/administration/smites.ftl [hunk 1]
```diff
@@ -44,7 +44,7 @@ admin-smite-reptilian-species-swap-description = It changes their species to Rep
 admin-smite-locker-stuff-description = Stuffs them in a (welded) locker.
 admin-smite-headstand-description = Vertically flips their sprite.
 admin-smite-plasma-internals-description = Replaces the contents of their internals with plasma.
-admin-smite-become-mouse-description = They become a rat. Ratatouille.
+admin-smite-become-mouse-description = They become a mouse. Squeek!
```
Comment (deltanedas) [inline Resources/Locale/en-US/administration/smites.ftl:47]: Squeak

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by rosieposieeee** [comment]
this is a great change!

**Comment by baddiePRO** [comment]
can't wait to go into maints to see to see all the lights dead and 20 pairs of red eyes looking at me

**Comment by VasilisThePikachu** [comment]
> 🆑
> 
>     * tweak: Ratkings and Rat servants eyes now glow in the dark
> 
>     * admin: Fixed smite text from Rat to Mouse.

Just for feature reference this is not how you do admin changelogs
you would just add ADMIN: underneath the :cl:

However, the changelog bot does not have support to make both an admin and regular changelog in one pr. So we can remove that.

Otherwise very nice

**Pull request merged**